samples/subsys/usb/mass/boards/rpi_pico.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 Kelly Helmut Lord
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &code_partition;
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		code_partition: partition@100 {
+			label = "code";
+			reg = <0x100 (DT_SIZE_M(1) - 100)>;
```

Comment:
This looks suspicious, https://github.com/zephyrproject-rtos/zephyr/blob/e364f29bffdfb122fead4b7dad082428f2f6969c/boards/arm/rpi_pico/rpi_pico-common.dtsi#L86-L87

0x100? 256? 100?

Reply:
Added reference to RP2040 datasheet for this. Also changed to DT_SIZE_M(2) - 0x100, this was an error.